src/ray/raylet/node_manager.cc

```diff
@@ -652,14 +674,19 @@ void NodeManager::HeartbeatAdded(const ClientID &client_id,
 void NodeManager::HeartbeatBatchAdded(const HeartbeatBatchTableData &heartbeat_batch) {
   const ClientID &local_client_id = gcs_client_->client_table().GetLocalClientId();
   // Update load information provided by each heartbeat.
+  std::unordered_set<ObjectID> active_object_ids;
   for (const auto &heartbeat_data : heartbeat_batch.batch()) {
+    for (int i = 0; i < heartbeat_data.active_object_id_size(); i++) {
+      active_object_ids.insert(ObjectID::FromBinary(heartbeat_data.active_object_id(i)));
```

Comment:
this `active_object_ids` is not used?

Reply:
if this is not used in this PR, please add a TODO here?

Reply:
Right, this isn't used yet. Will be used in a follow-up. Adding a TODO.